Syntaxes/RSpec.tmLanguage

```diff
@@ -92,7 +94,7 @@
 		<key>behaviour</key>
 		<dict>
 			<key>begin</key>
-			<string>^\s*([fx]?(?:describe|context|feature))\b(?=.*\bdo\s*$)</string>
+			<string>^\s*([fx]?(?:RSpec\.describe|describe|context|feature))\b(?=.*\bdo\s*$)</string>
```

Comment:
This won’t work for `RSpec.fdescribe` and `RSpec.xdescribe` (but it would match e.g. `fRSpec.describe`, which doesn’t make sense). 

As far as I understand, all methods mentioned in the “Defining Example Groups” section of http://rspec.info/documentation/3.5/rspec-core/RSpec/Core/ExampleGroup.html can be used at the outermost level, so the grammar should accept the `RSpec.` prefix for all of them … (I might be wrong, though – what do you think?)